code/modules/power/lights.dm

```diff
@@ -769,7 +773,7 @@
 		update_icon()
 		return
 	emergency_mode = TRUE
-	set_light(3, 1.7, bulb_emergency_colour)
+	set_light((fitting == "tube" ? 3 : 2), 1, bulb_emergency_colour)
```

Comment:
Is there no define for this?

Reply:
nope they use text :) (light tube code is so bad, i nearly died making floor lights)